src/main/java/picard/sam/AbstractAlignmentMerger.java

```diff
@@ -574,8 +574,8 @@ public static void fixNmMdAndUq(final SAMRecord record, final ReferenceSequenceF
      */
     public static void fixUq(final SAMRecord record, final ReferenceSequenceFileWalker refSeqWalker, final boolean isBisulfiteSequence) {
         if (record.getBaseQualities() != SAMRecord.NULL_QUALS) {
-            final ReferenceSequence refSeq = refSeqWalker.get(refSeqWalker.getSequenceDictionary().getSequenceIndex(record.getReferenceName()));
-            final byte[] referenceBases = refSeq.getBases();
+            int seqIndex = refSeqWalker.getSequenceDictionary().getSequenceIndex(record.getReferenceName());
```

Comment:
actually...I'm really confused by this...why not get the reference index directly from the record?